modules/logging-create-clf.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-multi-clf.adoc
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: REFERENCE
+[id="logging-create-clf_{context}"]
+= Creating a log forwarder
+
+include::snippets/logging-about-clf.adoc[]
+
+If using multiple log forwarders is enabled in your cluster, you can create `ClusterLogForwarder` custom resources (CRs) in any namespace, using any name.
```

Comment:
This again speaks to the "legacy" or "multi CLF" mode from the enhancement doc.  There really is no "enablement". It is a matter of creating: SA, rolebindings, CLF in a given namespace such that it is not named openshift-logging/instance.

* legacy = CL/CLF: openshift-logging/instance
* multi = CLF: <any>/<any> but not openshift-logging/instance